src/app/login/oauth2/code/google/page.tsx

```diff
@@ -1,28 +1,41 @@
 'use client';
-import React, { useEffect, useState } from 'react';
-import { useSearchParams } from 'next/navigation';
+import React, { useEffect } from 'react';
```

Comment:
여기 React는 없어도 될 거 같습니다~!
```suggestion
import { useEffect } from 'react';
```

Reply:
반영 완료~~!